docs/src/_includes/components/npx_tabs.macro.html

```diff
@@ -0,0 +1,46 @@
+{%- macro npx_tabs(params) -%}
+<div class="c-tabs" data-tabs>
+   <div class="c-tabs__tablist" data-tablist hidden>
+      <button class="c-tabs__tab" data-tab>npm</button>
+      <button class="c-tabs__tab" data-tab>yarn</button>
+      <button class="c-tabs__tab" data-tab>pnpm</button>
+      <button class="c-tabs__tab" data-tab>bun</button>
+   </div>
+   <div class="c-tabs__tabpanel" data-tabpanel>
+      <h3 class="h6 c-tabs__tabpanel__title">npm</h3>
+
+```shell
+{% if params.comment %}# {{ params.comment | safe }}{{ "\n" }}{% endif -%}
```

Comment:
check if we can optional chain on comment here? We have to pass comment as null everytime we don't have any comments which seems un necessary

Reply:
Sorry, didn't got you.
Can you specify what change in the code you want to do?

Reply:
I think he mean we can use optional chaining `params?.comment`.

But i tried the existing code locally and i am not getting any errors on removing `comment: null`  property.

@Jay-Karia, can you please try to compile the code without `comment: null`, let's see if it is working for you also or not.

Reply:
@Tanujkanti4441 ,
Tried on my machine, it worked too

Did you mean `params.comment?`

Reply:
No, i mean let's try the code we have right now without `comment: null`, as you said you are getting errors let's check that again.

Reply:
> @Tanujkanti4441 ,
> Tried on my machine, it worked too

Great, then we can safely remove the `comment: null` from the docs.

> Did you mean `params.comment?`

No, i mean `params?.comment`, but if the code is working with the `if statement` then we can leave this approach for now.

Reply:
> if the code is working with the if statement then we can leave this approach for now.

Should I resolve this conversation ?

Reply:
If we don't need `comment: null` can you please remove them from the docs, i think that's what this suggestion targets?

Reply:
I also tried to run it without comment and didn't break. So its safe to assume that removing `comment: null` from every `npx_tabs` would'nt break. Please once confirm and push related change. Lets close this

Reply:
Yeah, will do it soon.